tests/default/_core/search/aggs/nested_agg.yaml

```diff
@@ -0,0 +1,96 @@
+$schema: ../../../../../json_schemas/test_story.schema.yaml
+
+description: Test aggregation.
+prologues:
+  - path: user_infos
+    method: PUT
+    request_body:
+      payload:
+        mappings:
+          properties:
+            user:
+              type: nested
+              properties:
+                name:
+                  type: keyword
+                address:
+                  type: nested
+                  properties:
+                    city:
+                      type: nested
+                      properties:
+                        country:
+                          type: keyword
+  - path: /_bulk
+    method: POST
+    request_body:
+      content_type: application/x-ndjson
+      payload:
+        - {index: {_index: user_infos, _id: '0'}}
```

Comment:
Could we keep with the theme of movies and books? We plan to use these in the documentation, trying to be a bit more consistent.